addons/project/views/project_portal_templates.xml

```diff
@@ -85,6 +85,7 @@
 
     <template id="portal_my_project" name="My Project">
         <t t-call="portal.portal_layout">
+            <t t-set="title" t-value="project.name + ' | My Website'"/>
```

Comment:
Same comment than your changes in portal module, I don't think it is a good idea to do that since the `My Website`, I guess it is the default name of the website in demo data. 🤔 

And we don't have website module in `project` module